guide/interactions/slash-commands.md

```diff
@@ -476,3 +476,42 @@ await interaction.reply('Pong!');
 const message = await interaction.fetchReply();
 console.log(message);
 ```
+
+## Slash command permissions
+
+Slash commands have their own permissions system, which allows you to set the required permissions in order to use a command.
+
+::: tip
+The slash command permissions for guilds are only defaults and can be altered by guild administrators.
+:::
+
+### DM permission
+
+You can use the `setDMPermission` method to control if a global command can be used in DMs. By default, all global commands can be used in DMs.
+
+```js {6}
+const { SlashCommandBuilder } = require('@discordjs/builders');
+
+const data = new SlashCommandBuilder()
+	.setName('boop')
+	.setDescription('Replies with beep!')
+	.setDMPermission(false);
+```
+
+### Member permissions
+
+You can use the `setDefaultMemberPermissions` method to set the default permissions required by a member in order to run the command, setting it to `0` will prohibit anyone in a guild from using the command unless a specific overwrite is configured or the user has admin permissions.
+
+```js {9}
+const { SlashCommandBuilder } = require('@discordjs/builders');
+const { Permissions } = require('discord.js');
+
+const data = new SlashCommandBuilder()
+	.setName('ban')
+	.setDescription('Ban a member!')
+	.addUserOption(option =>
+		option.setName('target').setDescription('The member to ban'))
+	.setDefaultMemberPermissions([Permissions.FLAGS.KICK_MEMBERS, Permissions.FLAGS.BAN_MEMBERS]);
```

Comment:
```suggestion
	.setDefaultMemberPermissions(new Permissions([Permissions.FLAGS.KICK_MEMBERS, Permissions.FLAGS.BAN_MEMBERS]).bitfield);
```

Reply:
Wasn't the original point of builders to be able to be independent of discord.js, and usable for raw REST calls? Or at least only have a reliance on types?

Couldn't we represent this using something like 
```js
const { PermissionFlagsBits } = require("discord-api-types/v10");
// and
.setDefaultMemberPermissions(PermissionFlagsBits.KickMembers | PermissionFlagsBits.BanMembers)
```

Reply:
Maybe we'd have to, but I'd still prefer that as builders are supposed to be agnostic. Maybe it would be better if the builder method itself could accept an array of bits, but I don't think it does.

Reply:
It doesn't unfortunately, ill go with the bits method then and add a docs link to mdn with some pretext and a tip